browser/resources/settings/brave_overrides/people_page.js

```diff
@@ -32,6 +32,18 @@ RegisterPolymerTemplateModifications({
       console.error('[Brave Settings Overrides] People Page cannot find signin template')
       return
     }
+    const syncSetupLink = templateContent.querySelector('#sync-setup')
+    if (syncSetupLink) {
+      syncSetupLink.parentNode.removeChild(syncSetupLink)
```

Comment:
Is there a reason why not `syncSetupLink.remove()` for all of these?

Reply:
Oh, I didn't know `remove()` :) All replaced with it. thanks!